.circleci/config.yml

```diff
@@ -136,7 +136,7 @@ jobs:
     - run:
         name: Install helm
         environment:
-          HELM_VERSION: v3.0.0-rc.2
+          HELM_VERSION: v3.0.0
```

Comment:
I think this resulted in the integration test to fail:

![image](https://user-images.githubusercontent.com/22009/68856405-c1821900-0723-11ea-9591-4b7cb9645835.png)

Actually, I've also encountered this while bumping this along with the helm version in Dockerfile.

The fix was https://github.com/roboll/helmfile/pull/965/files#diff-452b35dd32cfe6d2251b4e8a4f7e1fc0R49

Reply:
Ah, thanks for the fix!

The integration test actually failed once before I made that change -- I tried bumping to 3.0.0 but was nearly midnight and I missed quite a few places :stuck_out_tongue: I'll rebase onto #965.